app/commands/assigning.rb

```diff
@@ -1,4 +1,6 @@
 module Assigning
+  StateHasChanged = Class.new(StandardError)
```

Comment:
This syntax is interesting to me why not:

`class StateHasChanged < StandardError end` this is the more common way I've seen errors defined. We could add all the errors to a seperate file as well if we wanted one place for them

Reply:
Will do.